src/components/range-date-picker/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@bcrs-shared-components/range-date-picker",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "lodash": "4.17.21"
+  },
+  "devDependencies": {
+    "vue-facing-decorator": "^3.0.3"
```

Comment:
Looks like it doesn't depend on  lodash or vue-facing-decorator.

Looks like it depends on:
- vue
- shared base-date-picker